examples/hono-react-vercel-edge/vite.config.js

```diff
@@ -10,7 +10,7 @@ export default {
       entry: {
         index: 'server/node-entry.js',
         app: {
-          path: 'server/app.js',
+          entry: 'server/app.js',
```

Comment:
I'm surprised it was working before. Maybe the API is inconsistent, e.g. the follow error message seems inconsistent: https://github.com/vikejs/vike-node/blob/15a298ea4cf4cbab31abb4e5776ac0c8253348c8/packages/vike-node/src/plugin/plugins/serverEntryPlugin.ts#L40.